src/app/components/calendar/calendarForm.tsx

```diff
@@ -25,34 +26,27 @@ const CalendarForm = () => {
     color: "blackAndWhite",
   };
 
-  const [calendarSetup, setCalendar] = useState<CalendarValuesProps | null>(null);
-
-  const [calendarDates, setCalendarDates] = useState({
+  const calendarDatesDefault = {
     weekNumber: null,
     weekDayNumbers: [],
     month: new Date().getMonth(),
     year: new Date().getFullYear(),
-  });
+  }
+
+  const [calendarSetup, setCalendar] = useLocalStorage('calendarData', calendarDefaultValues)
+  const [calendarDates, setCalendarDates] = useLocalStorage('calendarDatesData', calendarDatesDefault)
+
+  const [mount, setMount]= useState(false)
 
   // TMP
-  const handleInput = (e: any) => {
+  const handleInput = (e: { target: { [key: string]: any }}) => {
```

Comment:
```suggestion
  const handleInput = (e: React.MouseEvent) => {
```

Overall, I see plenty of `any` 😅